cli/ops/net.rs

```diff
@@ -179,7 +179,8 @@ fn op_shutdown(
   let shutdown_mode = match how {
     0 => Shutdown::Read,
     1 => Shutdown::Write,
-    _ => unimplemented!(),
+    2 => Shutdown::Both,
```

Comment:
This has a different name in typescript. Would it make sense to unify rust to be `Shutdown::ReadWrite`?